mne/preprocessing/maxwell.py

```diff
@@ -157,6 +162,29 @@ def maxwell_filter_prepare_emptyroom(
     )
 
     raw_er_prepared = raw_er.copy()
+    # just in case of combine MEG/other modality, let's explicitly drop other channels
+    # that might have a digitization and emit a warning.
+    picks = [elt for elt in ("eeg", "ecog", "seeg", "dbs") if elt in raw_er_prepared]
+    if len(picks) != 0:
+        picks = _picks_to_idx(raw_er_prepared.info, picks=picks, exclude=())
+        idx = np.setdiff1d(np.arange(raw_er_prepared.info["nchan"]), picks)
+        raw_er_prepared.pick(idx, exclude=())
+    if len(raw_er.ch_names) != len(raw_er_prepared.ch_names):
+        warn(
+            "The empty-room recording contained EEG-like channels. These channels "
+            "were dropped from the empty-room recording, as they are not "
+            "compatible with Maxwell filtering. If you need to, add those channels "
+            "back after the execution of 'maxwell_filter_prepare_emptyroom' from your "
+            "original empty-room recording using 'raw.add_channels'."
+        )
```

Comment:
The more I think about it, the more I feel like it should simply be `.pick("meg", exclude=())`.

Reply:
Not sure about adding the EEG channels back, but regarding `.pick("meg", exclude=())` there are other channel types that you could reasonably want to have around (e.g., stim, CHPI status, active shield status, etc.) and it's hard to know every end-users use case / needs, hence the use of a blocklist approach (eeg-like) here rather than an allowlist (meg) approach.